<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Wlanconfig</class>
 <widget class="QDialog" name="Wlanconfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Wlan</string>
  </property>
  <property name="windowIcon">
   <iconset resource="assets.qrc">
    <normaloff>:/assets/WiFi2.svg</normaloff>:/assets/WiFi2.svg</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="1,7">
   <property name="spacing">
    <number>3</number>
   </property>
   <property name="leftMargin">
    <number>5</number>
   </property>
   <property name="topMargin">
    <number>5</number>
   </property>
   <property name="rightMargin">
    <number>5</number>
   </property>
   <property name="bottomMargin">
    <number>5</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <property name="leftMargin">
      <number>5</number>
     </property>
     <property name="topMargin">
      <number>5</number>
     </property>
     <property name="rightMargin">
      <number>5</number>
     </property>
     <property name="bottomMargin">
      <number>5</number>
     </property>
     <item>
      <widget class="QPushButton" name="Btn_Back">
       <property name="whatsThis">
        <string>BTStyleSheet2</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="assets.qrc">
         <normaloff>:/assets/huitui.svg</normaloff>:/assets/huitui.svg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>72</width>
         <height>72</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="Btn_Logo">
       <property name="styleSheet">
        <string notr="true">border:none;
font: 75 30pt &quot;Gill Sans MT&quot;;
</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="assets.qrc">
         <normaloff>:/assets/Wifi.svg</normaloff>:/assets/Wifi.svg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>72</width>
         <height>72</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="Btn_Help">
       <property name="whatsThis">
        <string>BTStyleSheet2</string>
       </property>
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="assets.qrc">
         <normaloff>:/assets/help.svg</normaloff>:/assets/help.svg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>72</width>
         <height>72</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout" rowminimumheight="6,1" columnminimumwidth="7,1">
     <property name="spacing">
      <number>5</number>
     </property>
     <item row="0" column="0">
      <widget class="QListView" name="WPA_List">
       <property name="font">
        <font>
         <pointsize>30</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(64, 64, 64);</string>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,0">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QPushButton" name="Btn_DevRefresh">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="whatsThis">
          <string>BTStyleSheet2</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="assets.qrc">
           <normaloff>:/assets/shuaxin.svg</normaloff>:/assets/shuaxin.svg</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>72</width>
           <height>72</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <pointsize>21</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>Wlan Device :</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="DevList">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <pointsize>30</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 255, 255);</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="1" rowspan="2">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QPushButton" name="Btn_Connect">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="whatsThis">
          <string>BTStyleSheet2</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="assets.qrc">
           <normaloff>:/assets/wangluo.svg</normaloff>:/assets/wangluo.svg</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>72</width>
           <height>72</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="Btn_Scan">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="whatsThis">
          <string>BTStyleSheet2</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="assets.qrc">
           <normaloff>:/assets/sanweidianyunsaomiao.svg</normaloff>:/assets/sanweidianyunsaomiao.svg</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>72</width>
           <height>72</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="Btn_DevSwitch">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="whatsThis">
          <string>BTStyleSheet2</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="assets.qrc">
           <normaloff>:/assets/shebeiweilianjie-copy.svg</normaloff>:/assets/shebeiweilianjie-copy.svg</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>72</width>
           <height>72</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="assets.qrc"/>
 </resources>
 <connections/>
</ui>
